### PR TITLE
Feature/271 calendar

### PR DESCRIPTION
**Pull request by mkowalczyk97**
@weronikaolejniczak @CalderBriar 

Calendar feature done.

Configuration file is located in /src/CalendarEventChandler.js
Added Snackbar in order to give choice for user either to save to device's calendar or leave it as it is.

Cheers

### CODE REVIEW

File: src/CalendarEventChandler.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        console.log('Please Insert Event Id');
+      return;
+    }
+    const eventConfig = {
+      eventId,
+    };
+    
+    AddCalendarEvent.presentEventEditingDialog(eventConfig)
+      .then((eventInfo) => {
+        console.log('eventInfo -> ' + JSON.stringify(eventInfo));
```
Comment (CalderBriar) [inline src/CalendarEventChandler.js:53]: left console.log

File: src/CalendarEventChandler.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      })
+      .catch((error) => {
+        console.log('Error -> ' + error);
```
Comment (CalderBriar) [inline src/CalendarEventChandler.js:56]: another one

File: src/CalendarEventChandler.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      });
+  };
+
+  showCalendarEventWithId = (eventId) => {
+    if (!eventId) {
+        console.log('Please Insert Event Id');
```
Comment (CalderBriar) [inline src/CalendarEventChandler.js:62]: one more

**Review by CalderBriar — COMMENTED** [review]
Everything seems okay. 

Nevertheless I would consider changing some console.logs in "error catching" to actual pops for the user but more testing is required on my part.

Overall good job!

**Review by weronikaolejniczak — APPROVED** [review]
Looks good.

**Pull request merged**